bin/BuildPackages.sh

```diff
@@ -279,8 +248,8 @@ build_one_package() {
   ) || build_fail
 }
 
-date >> "$LOGDIR/$FAILPKGFILE.log"
-for PKG in "${PACKAGES[@]}"
+date >> "$LOGDIR/fail.log"
+for PKG in ${PACKAGES}
```

Comment:
Most unix shell scripts are horribly broken with regards to whitespace. E.g. the current BuildPackages.sh script is, too, because it incorrectly quotes in various places.

Anyway, I'll add back the arrays to make you guys happy, but again: this won't make it work with spaces in filenames, because other places in this script are doing it wrong (just try it)